### DISCUSSION TITLE
config.yml edits

### DISCUSSION THREAD

**Discussion by RonRe** [indrawp/Setup-Guide#38] [Q&A]
Hey.
I'm trying to edit the config.yaml according to the comments in it. I think I did it successfully (we'll see when I try to get the server to work), but there was one thing that I'm having trouble with.
In the file, there is an entry for replicationPassword and  replicationCaCertificate. What are those? I didn't see anything in the AWS documentation regarding the setup of these.
Please help

**Comment by madeindra** [comment]
@RonRe replicationPassword is just password, you create it as you see it fits. replicationCaCertificate it's CA Certificate, the issuer of the SSL / HTTPS certificate.

**Reply by RonRe** [reply]
@madeindra yes, I assumed that's the thing. Is there any place that explains what/how to set it up further? Or what it is?

**Reply by madeindra** [reply]
@RonRe To be honest i've not find any, I just assume it's related to CDS. But you can get some idea by reading the code, Signal server code is pretty easy to understand in my experience.
